BiometricLoginKotlin/app/src/main/java/com/example/biometricloginsample/BiometricPromptUtils.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.biometricloginsample
+
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import androidx.biometric.BiometricPrompt
+import androidx.core.content.ContextCompat
+
+// Since we are using the same methods in more than one Activity, better give them their own file.
+object BiometricPromptUtils {
+    private val TAG = "BiometricPromptUtils"
+    fun createBiometricPrompt(
+        activity: AppCompatActivity,
+        processSuccess: (BiometricPrompt.AuthenticationResult) -> Unit
+    ): BiometricPrompt {
+        val executor = ContextCompat.getMainExecutor(activity)
+
+        val callback = object : BiometricPrompt.AuthenticationCallback() {
```

Comment:
Callback can be created in `BiometricPromptUtils`.

Reply:
Please elaborate

Reply:
The `processSuccess` method is defined by the caller. See line 48